q2cli/cli.py

```diff
@@ -25,17 +25,24 @@ class QiimeCLI(click.MultiCommand):
     _builtin_commands = {'tools': q2cli._tools.tools,
                          'info': q2cli._info.info}
 
+    @property
+    def _name_map(self):
+        return {k.replace('_', '-'): k for k in self._plugin_manager.plugins}
```

Comment:
Can you replace ``k.replace('_', '-')`` with ``_name_to_command(k)``, so you're not duplicating the renaming logic here?